.github/workflows/android-tests.yml

```diff
@@ -0,0 +1,114 @@
+name: Tests
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  Android_tests:
+    if: github.event.review && (github.event.review.state == 'approved' || contains(github.event.review.body, '/check'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Preparing SSH key
+        run: |
+          CURRENT_DIR=$(pwd)
+          cd ~
+          mkdir .ssh && cd .ssh
+          echo ${{ secrets.SSH_KEY }} | base64 -d > id_rsa
+          sudo chmod 600 $(pwd)/id_rsa
+          cd $CURRENT_DIR
+
+      - name: Installing Rust toolchain
+        run: |
+          rustup toolchain install 1.41.0 
+          rustup install 1.41.0 
+          rustup target add wasm32-unknown-unknown --toolchain 1.41.0
+          rustup default 1.41.0
+          rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
+
+      - name: Installing WASM
+        run: |
+          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+
+      - name: Installing Python 2.7
+        run: |
+          sudo apt-get install python2 -y
+          sudo ln -sf /usr/bin/python2 /usr/bin/python
+      
+      - name: Installing NDK 21.4.7075529
+        run: $ANDROID_SDK_ROOT/tools/bin/sdkmanager --sdk_root=$ANDROID_SDK_ROOT --install "ndk;21.4.7075529"
+
+      - name: Installing Gradle
+        run: sudo apt-get install gradle -y
+
+      - uses: actions/checkout@v3
+
+      - name: Reading .nvmrc
+        id: nvm
+        run: echo "NVMRC=$(cat .nvmrc)" >> $GITHUB_OUTPUT
```

Comment:
is `GITHUB_OUTPUT` a new thing? I don't familiar with it.

Reply:
Yeap, kinda new
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/